content-warehouse/src/main/java/contentwarehouse/v1/UpdateDocumentSchema.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package contentwarehouse.v1;
+
+// [START contentwarehouse_update_document_schema]
+
+import com.google.cloud.contentwarehouse.v1.DocumentSchema;
+import com.google.cloud.contentwarehouse.v1.DocumentSchemaName;
+import com.google.cloud.contentwarehouse.v1.DocumentSchemaServiceClient;
+import com.google.cloud.contentwarehouse.v1.DocumentSchemaServiceSettings;
+import com.google.cloud.contentwarehouse.v1.PropertyDefinition;
+import com.google.cloud.contentwarehouse.v1.TextTypeOptions;
+import com.google.cloud.contentwarehouse.v1.UpdateDocumentSchemaRequest;
+import com.google.cloud.resourcemanager.v3.Project;
+import com.google.cloud.resourcemanager.v3.ProjectName;
+import com.google.cloud.resourcemanager.v3.ProjectsClient;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+public class UpdateDocumentSchema {
+  public static void updateDocumentSchema() throws IOException, 
+        InterruptedException, ExecutionException, TimeoutException { 
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "your-project-id";
+    String location = "your-region"; // Format is "us" or "eu".
+    String documentSchemaId = "your-document-schema-id";
+    updateDocumentSchema(projectId, location, documentSchemaId);
+  }
+
+  public static void updateDocumentSchema(String projectId, String location, 
+        String documentSchemaId) throws IOException, InterruptedException,
+          ExecutionException, TimeoutException { 
+    String projectNumber = getProjectNumber(projectId);
+
+    String endpoint = "contentwarehouse.googleapis.com:443";
+    if (!"us".equals(location)) {
+      endpoint = String.format("%s-%s", location, endpoint);
+    }
+
+    DocumentSchemaServiceSettings documentSchemaServiceSettings = 
+             DocumentSchemaServiceSettings.newBuilder().setEndpoint(endpoint).build(); 
+
+    // Create the Schema Service Client 
```

Comment:
Add the client initializer comment.

// Initialize client that will be used to send requests. This client only needs to be created once, and can be reused for multiple requests.